### PR TITLE
chore: update storybook and simplify its setup

### PR DESCRIPTION
**Pull request by gcornut**
# General summary

- Update storybook
- Simplify its configuration (https://medium.com/storybookjs/declarative-storybook-configuration-49912f77b78)
- Improve theme switcher
Capture d’écran de 2020-01-17 13-02-27

# Check list

-   [x] (if has code) Add `need: review-frontend` label

-   [x] (if fix or feature) Add `need: test` label

Check out the [contribution guide] for general guidelines for submissions to the design system.

[contribution guide]: https://github.com/lumapps/design-system/blob/master/CONTRIBUTING.md
[react dev check list]: https://github.com/lumapps/design-system/blob/master/CONTRIBUTING.md#react-developpment-check-list

### CODE REVIEW

**Comment by gcornut** [comment]
@juanigalan91 let me know what you think of these improvements (and if you also have time to test these it would be nice ^^)

File: packages/lumx-react/.storybook/main.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
```
Comment (juanigalan91) [inline packages/lumx-react/.storybook/main.js]: looks awesome ! extremely simple :)

**Comment by juanigalan91** [comment]
@gcornut I tried it out, works like a charm! Great work 💪🏻

However, I do feel that the themes (Material and Lumapps) and the light/dark are two different things, managed in different states and change different look and feel on the page. Even at a code level we are managing them separately, so from a developer experience, it feels a little bit weird to have these two things with the same UI and next to each other, since they look like one "radio button" group.

In any case, nothing blocking, I just wanted to point that out :)

**Comment by matmkian** [comment]
I agree, at least a little label in front of chip groups like on the demo site
Capture d’écran 2020-01-16 à 08 59 33

**Comment by gcornut** [comment]
> @gcornut I tried it out, works like a charm! Great work 💪🏻
> 
> However, I do feel that the themes (Material and Lumapps) and the light/dark are two different things, managed in different states and change different look and feel on the page. Even at a code level we are managing them separately, so from a developer experience, it feels a little bit weird to have these two things with the same UI and next to each other, since they look like one "radio button" group.
> 
> In any case, nothing blocking, I just wanted to point that out :)

Ok, I see. The main thing I wanted to acheive is to have both "theme" switcher on at the top so that hey both stay in place on every story. But yeah we could add labels and have the dark/light theme switcher use the `Switch` component like before

**Comment by matmkian** [comment]
Test ko -> when i switch from a story to another, no more style :-(

**Comment by matmkian** [comment]
Test KO -> les thèmes sont inversés et rien dans les stories d'autocomplete.

**Pull request merged**